src/device.rs

```diff
@@ -51,6 +56,40 @@ impl Device {
         }))
     }
 
+    /// Encrypt a to-device message to be sent to this device, using Olm
+    /// encryption.
+    ///
+    /// Prior to calling this method you must ensure that an olm session is
+    /// available for the target device. This can be done by calling
+    /// {@link OlmMachine.getMissingSessions}.
+    ///
+    /// The caller is responsible for sending the encrypted
+    /// event to the target device. If multiple messages are
+    /// encrypted for the same device using this method they should be sent in
+    /// the same order as they are encrypted.
+    ///
+    /// # Returns
+    /// Returns a Promise for a js object of the encrypted event.
+    /// Can be used to create the payload for a `/sendToDevice` API.
+    #[wasm_bindgen(js_name = "encryptToDeviceEvent")]
+    pub fn encrypt_to_device_event(
+        &self,
+        event_type: String,
+        content: JsValue,
+    ) -> Result<Promise, JsError> {
+        let me = self.inner.clone();
+        let content: Value = serde_wasm_bindgen::from_value(content)?;
+        let event_type = event_type.clone();
```

Comment:
This line can be removed. No need to clone the event type here, you already own it.

Reply:
```suggestion
```